zipline/lib/_windowtemplate.pxi

```diff
@@ -138,6 +145,8 @@ cdef class AdjustedArrayWindow:
         new_out = asanyarray(self.data[anchor - self.window_length:anchor])
         if view_kwargs:
             new_out = new_out.view(**view_kwargs)
+        if self.rounding_places is not None and new_out.dtype is dtype('float64'):
```

Comment:
Seems like `np.issubdtype` might be the more robust way to do this? Also any reason not to just import `float64` directly?